ctdcal/process_ctd.py

```diff
@@ -1065,6 +1065,8 @@ def calibrate_param(param,ref_param,press,ssscc,btl_num,xRange=None,):
 
     # return coef,df_ques
 
+# MK: deprecated 05/12/20
+# use fit_ctd._flag_btl_data() instead
 def quality_check(param,param_2,press,ssscc,btl_num,find,thresh=[0.002, 0.005, 0.010, 0.020]):
```

Comment:
Maybe use a real [deprecation warnings](https://docs.python.org/3/library/warnings.html#warnings.warn)? Though I guess it depends on how certain you are that this code isn't being called anymore... when it is at the point of others using this library, using actual warnings will probably be needed.

Reply:
Everything deprecated now should be cleaned up (99% removed, 1% maybe kept with proper deprecation warnings) before we hit public "1.0". Goal is to do an in person cleaning of code when I return to SD and then make the library much cleaner.